README.md

```diff
@@ -1,14 +1,218 @@
-# Robust-UI
-This component library is made to ease the work & to complete different projects using various components in this cl which are:-
-Avatar,
-Alert,
-Badge,
-Button,
-Card,
-Grid,
-Image,
-Input,
-List,
-Navigation,
-Rating,
-Toast.
+## Get Started
+
+To start using the components in your project, you can add the below `<link>` inside your `<head>` section of your page. Make sure to keep this as your first CSS import, if you would be using your own custom CSS classes.
```

Comment:
```suggestion
To start using the components in your project, you can add the below `<link>` inside the `<head>` section of your page. Make sure to keep this as your first CSS import, to avoid overriding your own css.
```